src/types/BlockHeader.zig

```diff
@@ -0,0 +1,42 @@
+const std = @import("std");
+
+pub const BlockHeader = struct {
```

Comment:
you can remove the struct declaration and the file will be your struct
https://zig.news/gowind/zig-files-are-structs-288j